diff -Naur pantheon-greeter-2.0.0.354.39.orig/CMakeLists.txt pantheon-greeter-2.0.0.354.39/CMakeLists.txt
--- pantheon-greeter-2.0.0.354.39.orig/CMakeLists.txt	2015-04-07 04:50:17.000000000 +0700
+++ pantheon-greeter-2.0.0.354.39/CMakeLists.txt	2015-06-10 23:19:16.780826705 +0700
@@ -67,4 +67,4 @@
 
 install (TARGETS pantheon-greeter RUNTIME DESTINATION sbin)
 install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/pantheon-greeter.desktop DESTINATION share/xgreeters)
-install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/pantheon-greeter.conf DESTINATION ${CONF_DIR})
+#install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/pantheon-greeter.conf DESTINATION ${CONF_DIR})
